libs/features/commerce/test/inlinePrice.test.html.js

```diff
@@ -0,0 +1,173 @@
+import Log from '../src/log.js';
```

Comment:
@yesil 
this is because this file is loaded from **html** test in `inlinePrice.test.html` and should be never selected by web test runner for direct execution (because requires hosting `html` file)
per current configuration, web test runner selects all files matching pattern `./test/**/*.test.(js|html)`
`.test.html.js` does not match this pattern